src/Lib/2D/CachedFontRenderer.cpp

```diff
@@ -98,3 +106,29 @@ void CachedFontRenderer::cleanup() {
   lastCleanedTick = currentTick;
   LOGGER.debug("Cached font cleanup: Done.");
 }
+
+#else
+
+#include "test.hpp"
+#include "CachedFontRenderer.hpp"
+
+void CachedFontRenderer::test_openFont(void) {
+  assert (openFont() != NULL);
```

Comment:
This test is failing on my system even though it appears correctly:

```
stderr:
2024-01-26 23:55:25 font path: /home/andy/src/netpanzer/netpanzer/data/fonts/GNUUnifont9FullHintInstrUCSUR.ttf
```

I'm still investigating.

Reply:
Fixed.